cross/znc/Makefile

```diff
@@ -1,28 +1,31 @@
 PKG_NAME = znc
-PKG_VERS = 1.8.2
+PKG_VERS = 1.9.1
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://znc.in/releases/archive
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
 ifeq ($(strip $(PYTHON_PACKAGE)),)
-PYTHON_PACKAGE = python311
+PYTHON_PACKAGE = python312
 endif
 
-BUILD_DEPENDS = cross/python311
-DEPENDS = cross/openssl3 cross/libicu cross/zlib
+BUILD_DEPENDS = cross/python312
+DEPENDS = cross/openssl3 cross/libicu cross/zlib cross/argon2
 
-# configure: No compiler with C++11 support was found
-# error: Upgrade your compiler. GCC 4.8+ and Clang 3.2+ are known to work.
-UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(OLD_PPC_ARCHS)
+# Ensure C++11 compatibility
```

Comment:
This comment should be adjusted.
DSM 6 with gcc >= 4.8 supports C++11 too.

Reply:
now fixed, refers to c++17